tests/erc20/main.rs

```diff
@@ -9,17 +9,27 @@ pub mod common;
 pub mod erc20;
 
 use ahash::AHashMap;
+use alloy_primitives::B256;
 use common::test_execute_revm;
 use erc20::generate_cluster;
-use pevm::{EvmAccount, InMemoryStorage};
+use pevm::{EvmAccount, EvmCode, InMemoryStorage};
 use revm::primitives::{Address, TxEnv};
 
 #[test]
 fn erc20_independent() {
     const N: usize = 37123;
     let (mut state, txs) = generate_cluster(N, 1, 1);
     state.insert(Address::ZERO, EvmAccount::default()); // Beneficiary
-    test_execute_revm(InMemoryStorage::new(state, []), txs);
+
+    let mut bytecodes: AHashMap<B256, EvmCode> = AHashMap::new();
```

Comment:
Let's return `bytecodes` from `generate_cluster`?

Reply:
resolved in the latest commit